docs/reference/tags/product_search/index.md

```diff
@@ -83,6 +83,8 @@ It's possible to use any of the following attributes in the search tag itself, t
 * `subcategory` Will search for products which match the subcategory passed, this must use one of Fixers predefined subcategories, will accept either a single Subcategory or an array.
 * `page_size` Will determine how many results are shown per page. If this is not included it will deafult to 12 results per page. e.g. `page_size: 12` 
 * `include_organisation_products` This allows you to include products from other companies which are linked in the same organisation. e.g. `include_organisation_products: true`
+* `sort` The sorting order of results. The available options are: `name_asc`, `name_desc`, `duration_asc`, `duration_desc`, `departure_date_asc` and `departure_date_desc`.
```

Comment:
I know its pretty obvious what `asc` and `desc` mean, but can we assume all the readers of the docs will have that knowledge? Should we add something just to clarify?

Reply:
Yeah, good shout.. updated!